CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Add your own contributions to the next release on the line below this, please include your name too. Please don't set
   a new version if you are the first to make the section for `master`.
+* Add milestone model to issue and pull request. - d-date
```

Comment:
Not sure if we want to mention that we broke the api with `String` -> `Date`. The project is still quite young